### ISSUE TITLE
Roboto and PT Sans are not available on gh-pages branch

### ISSUE THREAD

**Issue by pathmapper** [openmaptiles/fonts#8]
If I understand the [README.md](https://github.com/openmaptiles/fonts/blob/master/README.md) correct, they should be available. 

![image](https://user-images.githubusercontent.com/20856381/45931223-31a36d80-bf6b-11e8-80e9-bcacd26c09de.png)

**Comment by stevage** [comment]
Yup, came to report the same thing. The actual font names that are available are listed here:

https://github.com/openmaptiles/fonts/blob/gh-pages/fontstacks.json

**Comment by amenk** [comment]
If you build from the master branch, the Noto Sans fonts don't have the "Klokantech" prefix .. should gh-pages be updated?
